src/routes/auth/guards/auth.guard.ts

```diff
@@ -38,10 +40,26 @@ export class AuthGuard implements CanActivate {
     }
 
     try {
-      this.authRepository.verifyAccessToken(accessToken);
+      this.jwtRepository.verifyToken(accessToken, JwtAccessTokenPayloadSchema);
       return true;
     } catch {
       return false;
     }
   }
+
+  private getAccessToken(request: Request, tokenType: string): string | null {
```

Comment:
I moved this here as it is isolated to this guard. As we will likely pivot to cookies, this is temporary.